packages/http-client-csharp/.tspd/docs/customization.md

```diff
@@ -1,151 +1,212 @@
-# Customizing the generated code
+## Customizing the generated code
 
-The C# code generator supports various ways to customize the generated code. This section covers the common customization techniques you can use to adapt the generated code to your specific needs.
+### Make a model internal
 
-## Make a model internal
+Define a class with the same namespace and name as generated model and use the desired accessibility.
 
-Define a class with the same namespace and name as the generated model and use the desired accessibility.
+<details>
```

Comment:
This seems out of place, as do the others.

Reply:
These cause the details to be collapsed by default which makes it a bit easier to navigate through.